src/v/redpanda/application.cc

```diff
@@ -356,6 +356,7 @@ int application::run(int ac, char** av) {
                 // thus avoiding crashloops.
                 schedule_crash_tracker_file_cleanup();
                 app_signal.wait().get();
+                trigger_abort_source();
```

Comment:
there is an abort_source in app_signal already. can we use (or expand and use) that?

Reply:
I did begin with that thought but I ran into an issue. Invoking this new sharded abort_source via invoke_all() returns a future but I can't block on it in app_signal's abort_source call back. To workaround it, I have to gate on it in an async fiber and that seemed unnecessarily complex.